edb/pgsql/delta.py

```diff
@@ -4086,28 +4061,32 @@ def _alter_pointer_type(self, pointer, schema, orig_schema, context):
                 schema=orig_schema,
             )
 
+        tab = q(*old_ptr_stor_info.table_name)
+        target_col = old_ptr_stor_info.column_name
+        aux_ptr_table = None
+        aux_ptr_col = None
+
+        source_rel_alias = f'source_{uuidgen.uuid1mc()}'
+
         # There are two major possibilities about the USING claus:
         # 1) trivial case, where the USING clause refers only to the
```

Comment:
We've eliminated the trivial case (which never really fired), so update the comment?

Reply:
Maybe explaining that we could consider actually detecting/optimizing it in the future?